src/main/java/ua/com/danit/service/PostService.java

```diff
@@ -28,6 +28,10 @@ public Post getCurrentPost() {
     return postRepository.findAll().get(0);
   }
 
+  public Post getPostById(Long id) {
+    return postRepository.getOne(id);
```

Comment:
I guess it would be better to use findById method here, cause this method is public and may be called outside this class. Moreover it's name doesn't match with actual action - https://www.javacodemonk.com/difference-between-getone-and-findbyid-in-spring-data-jpa-3a96c3ff